assets/org.firestormviewer.FirestormViewer.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.firestormviewer.FirestormViewer</id>
+  <launchable type="desktop-id">org.firestormviewer.FirestormViewer.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LGPL-2.1</project_license>
+  <name>Firestorm Viewer</name>
+  <url type="homepage">https://www.firestormviewer.org/</url>
+  <summary>Client for accessing 3D virtual worlds</summary>
+  <description>
+    <p>This is the official build of Firestorm Viewer, re-packaged into a Flatpak with out the box voice support. This package is not supported by the Firestorm Project or Linden Labs. Please report issues through the linked issue tracker.</p>
```

Comment:
Nitpick: put app description first. Instead of saying it's an official build repackaged into Flatpak, use a disclaimer like `This wrapper is not verified by, affiliated with, or supported by xxx`.